doc/source/_static/js/assistant.js

```diff
@@ -139,7 +139,7 @@ function rayAssistant(event) {
 
         async function readStream() {
             try {
-                const response = await fetch('https://ray-assistant-public-bxauk.cld-kvedzwag2qa8i5bj.s.anyscaleuserdata.com/chat', {
+                const response = await fetch('https://ray-assistant-public-98zsh.cld-kvedzwag2qa8i5bj.s.anyscaleuserdata.com/chat', {
```

Comment:
why is this deployed as a new service now? why the domain changes?

this means past versions of ray docs will no longer be working?

Reply:
@KamenShah ?

Reply:
I'm working with AWS to understand why the load balancer is returning 503s and load balancer logs are not helpful.

Reply:
yup the previous link has the load balancer issues that Kamen is looking into and this new service (with no changes to the actual application) works just fine.